src/GenFu/Fillers/DateTimeNullableFillers.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace GenFu
+{
+    public class DateTimeNullableFiller : PropertyFiller<DateTime?>
```

Comment:
I agree that the code is the same block, however the implementation is different. It's best to keep the two filters separate. Inert can't be done due to the Template Type and it's wrong to make two Filters tightly coupled to each other. For example Calling the dateTime filter within the DateTimeNullable Filter.